tensorflow/python/ops/math_ops.py

```diff
@@ -2591,7 +2591,26 @@ def reduce_logsumexp_v1(input_tensor,
                                                 reduction_indices)
   keepdims = deprecation.deprecated_argument_lookup("keepdims", keepdims,
                                                     "keep_dims", keep_dims)
-  return reduce_logsumexp(input_tensor, axis, keepdims, name)
+  keepdims = False if keepdims is None else keepdims
```

Comment:
Why is the whole body inlined here instead of calling the non-deprecated version?

Reply:
Really sorry for that, was thinking it was the only solution (or helper function) yesterday at 4am. Fixing now.